src/pipelines.py

```diff
@@ -6,14 +6,16 @@
 from .loaders import ImageDetectionLoader
 from .steppy.base import Step
 from .models import Retina
+from .retinanet import DataDecoder
+from .postprocessing import SubmissionProducer
```

Comment:
@apyskir I think PredictionFormatter is a better name

Reply:
@jakubczakon OK, can be